### PR TITLE
Adding contributor for Complex-Number exercise

### PR DESCRIPTION
**Pull request by jagdish-15**
# pull request

This PR adds my name to the contributors list following my recent contribution to the Java track. My contribution includes improvements and refinements to the [Complex Number exercise](https://github.com/exercism/java/pull/2852), where I contributed to changing some methods in reference answers and in the boiler code for starting the exercise. Thank you for reviewing and considering my addition to the list of contributors!
---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/main/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/practice/complex-numbers/.meta/config.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "SleeplessByte",
     "Smarticles101",
     "sshine",
-    "Wasy18"
+    "Wasy18",
+    "jagdishdrp"
```
Comment (kahgoh) [inline exercises/practice/complex-numbers/.meta/config.json:18]: This list is alphabetically sorted. Could you please move your username entry to maintain the order?

**Review by kahgoh — CHANGES_REQUESTED** [review]

File: exercises/practice/complex-numbers/.meta/config.json (hunk 1 above)
Comment (jagdish-15) [reply exercises/practice/complex-numbers/.meta/config.json:18]: Sure! Thank you for your insights.

Comment (kahgoh) [reply exercises/practice/complex-numbers/.meta/config.json]: To be honest, I haven't tested, _but_ this should actually be your Github handle (according to the [documentation](https://exercism.org/docs/building/tracks/practice-exercises)).

Comment (jagdish-15) [reply exercises/practice/complex-numbers/.meta/config.json]: I am sorry for not reading this part of the documentation prior to this, I will change this to my GitHub handle. Thank you again!

Comment (jagdish-15) [reply exercises/practice/complex-numbers/.meta/config.json]: This time, I've included my GitHub handle, and I believe this should work.

**Review by kahgoh — APPROVED** [review]

**Pull request merged**